datastore/google/cloud/datastore/transaction.py

```diff
@@ -152,13 +153,19 @@ def Entity(*args, **kwargs):
 
     :type client: :class:`google.cloud.datastore.client.Client`
     :param client: the client used to connect to datastore.
+
+    :type options: `google.cloud.datastore.transaction.TransactionOptions`
```

Comment:
@chemelnucfin 

1. This class doesn't seem to exist?
1. In general we don't ask users to explicitly create protobuf objects, so you'd either need a wrapper class of your own, or you should do as we discussed before (just use `read_only=False` as the argument and create `TransactionOptions` protobuf for the user).
1. You should use `:class:` before a Sphinx type